src/CMakeLists.txt

```diff
@@ -78,7 +78,6 @@ if (${POISSON_SOLVER} STREQUAL "FFT" AND ${BACKEND} STREQUAL "OMP")
   message(STATUS "Using the FFT poisson solver with 2decomp&fft")
   set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake")
   find_package(decomp2d REQUIRED)
-  include_directories(${decomp2d_INCLUDE_DIRS})
```

Comment:
This line is required to set the include paths for the 2decomp modules when building x3d2